### ISSUE TITLE
Compilation warning

### ISSUE THREAD

**Issue by Mirraz** [lkundrak/dev86#1]
Building dev86-0.16.20 by gcc-4.8.2 I'v got this compilation warning:
dbprintf.c:110:13: warning: incompatible implicit declaration of built-in function ‘strlen’ [enabled by default]

**Comment by lkundrak** [comment]
Yeah, there's a lot of compilation warnings with newer toolchains. I'm not going to just shoot this particular one, but I'd gladly accept patches to deal with the warnings.

**Comment by scontini76** [comment]
If nobody is already working on it I will be happy to contribute on this issue. Please assign it to me.

**Comment by mfld-fr** [comment]
As said in #15, I suggest you to contribute rather to Jody Bruchon's fork.

**Comment by jbruchon** [comment]
These headers would need to be added to `bcc/dbprintf.c`:

``` c
#include <string.h>
#include <unistd.h>
```

This was fixed in my fork in April 2015: https://github.com/jbruchon/dev86/commit/4ed1997c5f80728fdc1f4d55238d1cd5eef46c48
